doc/api/cli.md

```diff
@@ -181,6 +181,19 @@ Make built-in language features like `eval` and `new Function` that generate
 code from strings throw an exception instead. This does not affect the Node.js
 `vm` module.
 
+### `--dns-order=order`
```

Comment:
nit: Can this be `--dns-result-order`? There are other orders in DNS that are relevant (in particular, the one of the DNS servers that are tried).

Reply:
sgtm